RNAwesomeCardIO.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+package = JSON.parse(File.read('package.json'))
+
+Pod::Spec.new do |s|
+  s.name             = "RNAwesomeCardIO"
+  s.version          = package["version"]
+  s.summary          = package["description"]
+  s.requires_arc     = true
+  s.license          = package["license"]
+  s.homepage         = package["homepage"]
+  s.authors          = package["author"]
```

Comment:
Hey @gauravds, not sure how this is going to work, we've seen some inconsistent behaviour when generating `RNAwesomeCardIO.podspec.json`. We had to change this to:
```
s.authors            = { package["author"]["name"] => package["author"]["email"] }
```

Reply:
Not sure about this, because package[author] doesn't have the name, same as email. It contains only one string. In general, people are doing hardcoded as `RNFirebase` https://github.com/invertase/react-native-firebase/blob/master/ios/RNFirebase.podspec